UnityProject/Assets/Scripts/UI/Systems/Lobby/Gender.cs

```diff
@@ -24,7 +24,7 @@ public enum BodyType //Now with cup Rests
 
 public enum GenderPronouns
```

Comment:
wait a second, what's this. Why do we need two Gender enums?

Reply:
GenderPronouns? It was unused as far as i could tell

Reply:
nuke it if it is really unused